inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp

```diff
@@ -50,22 +50,73 @@
 
 bool ngraph::pass::ConvertOpSet1ToLegacy::run_on_function(std::shared_ptr<ngraph::Function> f) {
-    ngraph::pass::Manager OpSet1ToLegacy;
+    ngraph::pass::Manager manager;
     std::vector<std::shared_ptr<ngraph::pass::PassBase> > transforms;
 
-#define NGRAPH_PASS(NAME, NAMESPACE) transforms.push_back(OpSet1ToLegacy.register_pass<NAMESPACE::NAME>());
-#include <transformations/convert_opset1_to_legacy/convert_opset1_to_legacy_tbl.hpp>
-#undef NGRAPH_PASS
+    manager.register_pass<ngraph::pass::ConstantFolding>();
+    manager.register_pass<ngraph::pass::ConvertReduceToPooling>();
+    manager.register_pass<ngraph::pass::ConvertMod>();
+    manager.register_pass<ngraph::pass::ConvertMinimum>();
+    manager.register_pass<ngraph::pass::ConvertSubtract>();
+    manager.register_pass<ngraph::pass::ConvertDivide>();
+    manager.register_pass<ngraph::pass::ConvertNegative>();
+    manager.register_pass<ngraph::pass::ConvertDepthToSpace>();
+    manager.register_pass<ngraph::pass::ConvertSpaceToDepth>();
+    manager.register_pass<ngraph::pass::ConvertConvolutions>();
+    manager.register_pass<ngraph::pass::BatchNormDecomposition>();
+    manager.register_pass<ngraph::pass::ConstantFolding>();
+    manager.register_pass<ngraph::pass::MulAddVerification>();
+    manager.register_pass<ngraph::pass::MulAddFusion>();
+    manager.register_pass<ngraph::pass::ConstantFolding>();
+    manager.register_pass<ngraph::pass::ConvertMatMulToFCorGemm>();
+    manager.register_pass<ngraph::pass::PullTransposeThroughFQUp>();
+    manager.register_pass<ngraph::pass::ConstantFolding>();
+    manager.register_pass<ngraph::pass::ConvFusion>();
+    manager.register_pass<ngraph::pass::FullyConnectedBiasFusion>();
+    manager.register_pass<ngraph::pass::ConstantFolding>();
+    manager.register_pass<ngraph::pass::ReshapeFullyConnected>();
+    manager.register_pass<ngraph::pass::ReshapeFullyConnectedFusion>();
+    manager.register_pass<ngraph::pass::Reshape1DOps>();
+    manager.register_pass<ngraph::pass::ConvertNormalizeL2WithMulToNormalizeIE>();
+    manager.register_pass<ngraph::pass::ConstantEltwiseReduction>();
+    manager.register_pass<ngraph::pass::ConvertMulAddToScaleShiftOrPower>();
+    manager.register_pass<ngraph::pass::ConvertMulOrAddFinally>();
+    manager.register_pass<ngraph::pass::ConstantFolding>();
+    manager.register_pass<ngraph::pass::ConvertBroadcastToTiles>();
+
+
+    auto anchor = manager.register_pass<ngraph::pass::GraphRewrite>();
+    anchor->add_matcher<ngraph::pass::ConvertNormalizeL2ToLegacyMatcher>();
+    anchor->add_matcher<ngraph::pass::ConvertHardSigmoidToLegacyMatcher>();
+    anchor->add_matcher<ngraph::pass::ConvertProposalToLegacyMatcher>();
+    anchor->add_matcher<ngraph::pass::ConvertTileToLegacyMatcher>();
+    anchor->add_matcher<ngraph::pass::ConvertLRNToLegacyMatcher>();
+    anchor->add_matcher<ngraph::pass::ConvertPadToLegacyMatcher>();
+    anchor->set_name("ngraph::pass::ConvertOpSet1ToLegacy");
```

Comment:
Why "ConvertOpSet1ToLegacy"? It is only a part of it.

Reply:
Is it only example of "merged" passes that we are going to have in ConvertToLegacy or is it only the first example and it will be further reorganized?

Reply:
It's just usage example. Ideally all transformations below also can be merged into single GraphRewrite pass